src/components/ui/checkbox.tsx

```diff
@@ -4,24 +4,38 @@ import * as React from "react";
 
 import { cn } from "@/lib/utils";
 
+import { Label } from "./label";
```

Comment:
We should not be updating ShadCN components. If we want a different version that clubs both, we make a component. 

It will be painfull to upgrade at a later stage if we keep editing shadcn components.